sapi/fpm/fpm/fpm_status.c

```diff
@@ -61,7 +61,10 @@ int fpm_status_export_to_zval(zval *status)
 
 	/* copy the scoreboard not to bother other processes */
 	scoreboard = *scoreboard_p;
-	struct fpm_scoreboard_proc_s procs[scoreboard.nprocs];
+	struct fpm_scoreboard_proc_s *procs = safe_emalloc(scoreboard.nprocs, sizeof(*procs), 0);
+	if (procs == NULL) {
+		return FAILURE;
```

Comment:
I'd suggest not to mix "semantics", `FAILLURE` is meant for `zend_result` type, like above, I would suggest to return just -1.

Reply:
What I have assumed here is that `fpm_status_export_to_zval()` is really supposed to return `zend_result` according to PHP's guidelines, but I have not yet submitted a PR that actually does this.